viewer/org.eclipse.birt.report.viewer/birt/WEB-INF/web.xml

```diff
@@ -255,4 +255,353 @@
 	</jsp-config>
 <context-param><param-name>org.eclipse.jetty.jsp.precompiled</param-name><param-value>true</param-value></context-param>
 
+<!--
```

Comment:
These additions to web.xml is done by the JSP pre-compiler during build and should not commited since they cause problems with the web viewer when BIRT is executed from Eclipse IDE.